config/notifiers.go

```diff
@@ -220,8 +224,13 @@ type DiscordConfig struct {
 	WebhookURL     *SecretURL                  `yaml:"webhook_url,omitempty" json:"webhook_url,omitempty"`
 	WebhookURLFile string                      `yaml:"webhook_url_file,omitempty" json:"webhook_url_file,omitempty"`
 
-	Title   string `yaml:"title,omitempty" json:"title,omitempty"`
-	Message string `yaml:"message,omitempty" json:"message,omitempty"`
+	Title                string `yaml:"title,omitempty" json:"title,omitempty"`
+	Message              string `yaml:"message,omitempty" json:"message,omitempty"`
+	TitleURL             string `yaml:"title_url,omitempty" json:"title_url,omitempty"`
+	SkipFields           bool   `yaml:"skip_fields,omitempty" json:"skip_fields,omitempty"`
```

Comment:
I think we should remove this as I haven't seen users asking for this in the past. It's much easier to add it later if we need. It's much harder to remove something once it's been added.

Reply:
I kinda agree. On the other hand... https://github.com/prometheus/alertmanager/pull/3821 wants to add them. So apparently users are asking for them. What do you think? I'm okay with both options. I'll let you guys - as the actual code owners - decide what direction to choose here.

Reply:
I'd side with George here unless we're explicitly using them I'd be in favour of removing them - I get when you're coming from on the customization approach but we're very short on hands in the Alertmanager, the less code we have to maintain the better.

My approach for this tends to be: Create an issue with the specific functionality you want to introduce, if we get enough support (number of reactions or comments on the issue) for it we can always add it.